todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/activities/main/adapter/ToDoListAdapter.java

```diff
@@ -80,9 +80,9 @@ public void onClick(View view) {
                 Snackbar.make(mapButton, "this should hide this item when clicked maybe", Snackbar.LENGTH_SHORT).show();
             }
             if (view.getId() == R.id.mapImageButton) {
-                Intent intent = new Intent(context, MapsActivity.class);
+                Intent intent = new Intent(view.getContext(), MapsActivity.class);
```

Comment:
isnt view context the same as container context?, toDoNameTextBox context?, dateTextBox context?